public/stylesheets/site/2.0/13-group-blurb.css

```diff
@@ -73,6 +73,7 @@ li.blurb:after, .blurb .blurb:after {
 
 .blurb dl.stats {
   float: right;
+  line-height: 1.4;
```

Comment:
Could you make this 1.429 to fit in a bit better with our [em scale](http://otwcode.github.io/docs/front_end_coding/em-scale.html)?

Reply:
And approved -- thank you!